docs/tutorials/build_a_node_in_rust.md

```diff
@@ -691,9 +688,46 @@ fn handle_ldk_event(..) {
 
 **Dependencies:** `ChannelManager`, `ChainMonitor`, `KeysManager`, `BroadcasterInterface`
 
-**References:** [`Event` docs](https://docs.rs/lightning/*/lightning/util/events/enum.Event.html), [LDK sample node event handling example](https://github.com/lightningdevkit/ldk-sample/blob/bc07db6ca4a3323d8718a27f85182b8157a20750/src/main.rs#L101-L240), [`EventHandler` docs](https://docs.rs/lightning/*/lightning/util/events/trait.EventHandler.html)
+**References:** [`Event` docs](https://docs.rs/lightning/*/lightning/util/events/enum.Event.html), [`EventHandler` docs](https://docs.rs/lightning/*/lightning/util/events/trait.EventHandler.html),[LDK sample node event handling example](https://github.com/lightningdevkit/ldk-sample/blob/bc07db6ca4a3323d8718a27f85182b8157a20750/src/main.rs#L101-L240)
+
+### Step 16. Initialize the `ProbabilisticScorer` 
+**What it's used for:** to find a suitable payment path to reach the destination.
 
-### 17. Initialize the `Persister`
+```rust
+let params = ProbabilisticScoringParameters::default();
+
+let scorer = ProbabilisticScorer::new(params, Arc:clone(&network_graph));
+```
+
+**Dependencies** `NetworkGraph`
+
+**Reference** [`ProbabilisticScorer` docs](https://docs.rs/lightning/*/lightning/routing/scoring/type.ProbabilisticScorer.html),
+[LDK sample node ProbabilisticScorer example](https://github.com/lightningdevkit/ldk-sample/blob/main/src/main.rs#L612)
```

Comment:
Let's link to a specific commit so the line number doesn't go stale as the file changes.